db/migrate/20150923042714_create_dogs.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDogs < ActiveRecord::Migration
+  def change
+    create_table :dogs do |t|
+      t.string  :name, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.